src/eventshub/src/component/SearchItem/SearchItem.tsx

```diff
@@ -0,0 +1,17 @@
+import { Hit as AlgoliaHit } from 'instantsearch.js/es/types';
+
+type SearchItemProps = {
+  hit: AlgoliaHit<{
+    name?: string;
+    venue_name?: string;
+  }>;
+};
+
+export function SearchItem({ hit }: SearchItemProps) {
+  return (
+    <article>
+      {hit.venue_name && <h3>{hit.venue_name}</h3>}
+      {hit.name && <h3>{hit.name}</h3>}
```

Comment:
I'll have to circle back to this in https://stanfordits.atlassian.net/browse/EIRH-62 and drill down on the returned data based on which view is displayed 😅